src/web/components/CMP.tsx

```diff
@@ -1,35 +1,43 @@
 import React, { useState, useEffect } from 'react';
-import {
-    shouldShow,
-    setErrorHandler,
-} from '@guardian/consent-management-platform';
-import { ConsentManagementPlatform } from '@guardian/consent-management-platform/lib/ConsentManagementPlatform';
-
-type Props = {
-    cmpUi: boolean; // A switch to decide if we show CMP or not
-};
+import loadable from '@loadable/component';
+
+const ConsentManagementComponent = loadable.lib(() =>
+    import(
+        '@guardian/consent-management-platform/lib/ConsentManagementPlatform'
+    ),
+);
 
-export const CMP = ({ cmpUi }: Props) => {
+export const CMP = () => {
     const [show, setShow] = useState(false);
 
     useEffect(() => {
-        if (shouldShow()) {
-            setShow(true);
-
-            // setErrorHandler takes function to be called on errors in the CMP UI
-            setErrorHandler((errMsg: string): void => {
-                const err = new Error(errMsg);
-
-                window.guardian.modules.sentry.reportError(err, 'cmp');
-            });
-        }
+        import(
```

Comment:
Just for my understanding (and prosperity) - this is a plain [dynamic import](https://webpack.js.org/guides/code-splitting/#dynamic-imports) causing the code split on 'vendors' for consent platform (answering @oliverlloyd's question on why it's split into three).

This is what we can do to dynamically import and split libraries vs using `loadable` which is basically some nice sugar and API around dynamic imports (and thus Webpack treats both the same RE code splitting?)

Reply:
Although there are methods in loadable to import `lib` files - when would we use plain dynamic imports vs loadable?

Reply:
yes this is the classic webpack dynamic import, I was planning on using exclusively `loadable-components` but as you can see I am importing functions and not react components.

In `loadable-components` you can split out components using the classic way:
https://loadable-components.com/docs/component-splitting/

or split out libraries using HOCs:
https://loadable-components.com/docs/library-splitting/

I cannot use either within `useEffect` so I decided to instead implement webpack's dynamic import. I think both are ok but personally if it is not a react component I think webpack's dynamic imports are preferable